django_celery_beat/models.py

```diff
@@ -276,8 +277,8 @@ class CrontabSchedule(models.Model):
         max_length=64, default='*',
         verbose_name=_('Day(s) Of The Week'),
         help_text=_(
-            'Cron Days Of The Week to Run. Use "*" for "all". '
-            '(Example: "0,5")'),
+            'Cron Days Of The Week to Run. Use "*" for "all", Sunday '
```

Comment:
this changes might new a new schema migration. can you try makemigrations and see anything generate and commit that as well?

Reply:
Good catch. Committed and squashed into my original commit for improving the helptext.